draft-ietf-scitt-architecture.md

```diff
@@ -232,6 +233,10 @@ The Envelope contains the identity of the Issuer and information about the Artif
 A Signed Statement is a COSE Envelope wrapped around a Statement, binding the metadata in the Envelope to the Statement.
 In COSE, an Envelope consists of a protected header (included in the Issuer's signature) and an unprotected header (not included in the Issuer's signature).
 
+Equivocation:
+
+: a state where it is possible for a Transparency Service to publish different, contradictory Signed Statements to different Issuers and Verifiers about the same Artifact {{EQUIVOCATION}}.
```

Comment:
Important nit.
Issuers create signed statements they register, which means the Transparency Service must not provide different results to different verifiers. But, it doesn't provide results _to_ Issuers, in this sense.

The TS can't, and shouldn't, prevent an Issuer from making updated statements as it learns more. 
On Monday, the artifact "is fine". On Wednesday, the Issuer learns of a new vulnerability and registers an updated statement that says the artifact "isn't fine". This is a valid scenario, yet it's not equivocation. 

However, the TS shouldn't be able to return different results of the same Signed Statement to different Verifiers.

I'm wondering if we should better summarize Non-equivocation, and let specific references clarify it in context.

And, do we need to specify Equivocation _and_ Non Equivocation?

Reply:
> Important nit. Issuers create signed statements they register, which means the Transparency Service must not provide different results to different verifiers. But, it doesn't provide results _to_ Issuers, in this sense.

I guess this is semantics but I had not really considered otherwise: the Issuer is going to have a TS client that processes the result with Signed and Transparent Statements to complete further processing or present otherwise for whatever it needs, right? (If I maintain an open source application or library, I presume I would want the URL or an ID to either of those upon registration to present on my application or library's web site or repo, no? Or am I really presuming and overloading what an Issuer is?)

> The TS can't, and shouldn't, prevent an Issuer from making updated statements as it learns more. On Monday, the artifact "is fine". On Wednesday, the Issuer learns of a new vulnerability and registers an updated statement that says the artifact "isn't fine". This is a valid scenario, yet it's not equivocation.
> 
> However, the TS shouldn't be able to return different results of the same Signed Statement to different Verifiers.
> 
> I'm wondering if we should better summarize Non-equivocation, and let specific references clarify it in context.

I am fine with that, any suggestions? I will mull it over.
 
> And, do we need to specify Equivocation _and_ Non Equivocation?

I only did it that way because the original source that derived this defined Equivocation (the unhappy path), not Non-equivocation (happy path), and definitely did not define the former with concepts and vocabulary of SCITT. That is the only reason I defined both. It seemed odd to have a long embedded definition of Equivocation in the definition of Non-equivocation, or the state of "not that." 😆

Reply:
> In [logic](https://en.wikipedia.org/wiki/Logic), equivocation ("calling two different things by the same name") is an [informal fallacy](https://en.wikipedia.org/wiki/Informal_fallacy) resulting from the use of a particular word/expression in multiple [senses](https://en.wikipedia.org/wiki/Word_sense) within an argument.[[1]](https://en.wikipedia.org/wiki/Equivocation#cite_note-Damer2008-1)[[2]](https://en.wikipedia.org/wiki/Equivocation#cite_note-2)

> It is a type of [ambiguity](https://en.wikipedia.org/wiki/Semantic_ambiguity) that stems from a phrase having two or more distinct [meanings](https://en.wikipedia.org/wiki/Meaning_(linguistics)), not from the grammar or structure of the sentence.[[1]](https://en.wikipedia.org/wiki/Equivocation#cite_note-Damer2008-1)

-- https://en.wikipedia.org/wiki/Equivocation